src/panels.cpp

```diff
@@ -1027,6 +1061,30 @@ static void draw_stats( avatar &u, const catacurses::window &w )
     mvwprintz( w, point( 25, 0 ), c_light_gray, _( "PER" ) );
     mvwprintz( w, point( stat < 10 ? 30 : 29, 0 ), stat_clr,
                stat < 100 ? to_string( stat ) : "99+" );
+
+    int weariness = u.weariness_level();
```

Comment:
I don't think these weariness levels belongs with stats, which is for DEX, STR, INT, and PER.  If you do it this way, when you also force the user to arrange the weariness bar under the stats bar for things to look correct.  Why not add a two-line weariness bar?

Reply:
As I stated in the commit adding these panels, I think the weariness and activity level displays are very important, while the stuff I included in the panel is a bit more extraneous, and would be something that would make a lot of sense to give up if you're short on log space.

I don't feel strongly this way, and am definitely open to changing it.

Reply:
I think that we should go with a two line weariness panel and default it to being enabled, but would prefer to see weariness added to the bottom of the Needs panel rather than the Stats panel if we want to make sure people see it.

I also want to support putting weariness anywhere in the side panel that the player wants, rather than splitting the basics into one of the current panels and then adding the details in a new panel.

Reply:
I was going to say I don't think it fits better in stats than in needs, but I don't actually find myself very convinced by that.

I think I may need some time to think this over, there are good arguments for both methods.